Zilon.Core/Zilon.Bot.Sdk/AvailableModesAttribute.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Zilon.Bot.Sdk
+{
+    [AttributeUsage(AttributeTargets.Property, Inherited = false, AllowMultiple = false)]
+    public sealed class AvailableModesAttribute : Attribute
+    {
+        public AvailableModesAttribute()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5310892)